src/react/context/ApolloProvider.tsx

```diff
@@ -22,6 +25,10 @@ export const ApolloProvider: React.FC<ApolloProviderProps<any>> = ({
           context = Object.assign({}, context, { client });
         }
 
+        if (suspenseCache && !context.suspenseCache) {
```

Comment:
```suggestion
        if (suspenseCache) {
```

No need to check this. Anytime a `suspenseCache` is provided, use it in `context`

Reply:
[34245f7](https://github.com/apollographql/apollo-client/pull/10323/commits/34245f748f1b15aef6ee8e66347d0c15a68c9363)